### PR TITLE
Fix/home page

### PR DESCRIPTION
**Pull request by boidushya**

### CODE REVIEW

File: lib/model/transfer/TransferCtrl.dart [hunk 1]
```diff
@@ -19,8 +19,6 @@ class TransferCtrl {
       print('No from address value specified');
       return false;
     }
-    // 5CdFxKLc95MxH21fYKEyWv7pnzL7xh5DENYFkVBxrhf9JSe1
-    // 5CdFxKLc95MxH21fYKEyWv7pnzL7xh5DENYFkVBxrhf9JSe1
     const toAddress = '5DWsQ5XpdixnPZUMZMiPuGRXaUKN115YNNqDcneqWfKaqvsK';
```
Comment (matjazonline) [inline lib/model/transfer/TransferCtrl.dart:22]: this address should not be hardcoded

File: lib/components/home/hero_video.dart [hunk 2]
```diff
@@ -0,0 +1,310 @@
...
+                  );
+                }
+              }),
+        ],
+      ),
+    );
+  }
+}
+
+// class VideoPlayerScreen extends StatefulWidget {
```
Comment (matjazonline) [inline lib/components/home/hero_video.dart:149]: we can remove comment

File: lib/pages/home_page.dart [hunk 3]
```diff
@@ -184,18 +189,37 @@ class _HomePageState extends State<HomePage> {
...
-            value: const SystemUiOverlayStyle(
-              statusBarColor: Colors.transparent,
-              statusBarIconBrightness: Brightness.dark,
-            ),
-            child: CustomScrollView(
-              physics: const BouncingScrollPhysics(),
-              clipBehavior: Clip.none,
-              slivers: [
-                /*Row(children: [
+    void openModal(String modalName) {
```
Comment (matjazonline) [inline lib/pages/home_page.dart:192]: we can share method with accounts page

File: pubspec.lock [hunk 4]
```diff
@@ -168,14 +168,21 @@ packages:
...
     dependency: transitive
     description:
       name: crypto
       url: "https://pub.dartlang.org"
     source: hosted
     version: "3.0.2"
+  csslib:
+    dependency: transitive
+    description:
+      name: csslib
```
Comment (matjazonline) [inline pubspec.lock:182]: do we really need another dependency

File: pubspec.lock [hunk 5]
```diff
@@ -387,6 +394,13 @@ packages:
       url: "https://pub.dartlang.org"
     source: hosted
     version: "1.1.3"
+  html:
+    dependency: transitive
+    description:
+      name: html
```
Comment (matjazonline) [inline pubspec.lock:400]: can we remove

**Comment by ghost** [comment]
hello guys, just wanted to say that you are awesome. i follow very closely the work you are doing for the mobile app version (i use it every day even if it's hard sometimes) and i see that you are working hard. don't worry about stress but simply work with peace of mind. in the future if you need any feedback before launching the app let me know, i would be happy to help you. good luck to everyone
CipAndrea

**Comment by matjazonline** [comment]
breaks navigation

**Pull request closed without merging**